rules/opentitan_test.bzl

```diff
@@ -310,17 +310,21 @@ def opentitan_functest(
         elif target in ["cw310_rom", "cw310_test_rom"]:
             devices_to_build_for.append("fpga_cw310")
 
+            # If the user passed in a bitstream, record it here so that we do
+            # not override it later
+            user_bitstream = cw310.get("bitstream") if cw310 else None
```

Comment:
I took a very quick look at the log files but should this line set `user_bitstream` to `None` also if `cw310["bitstream"]` is the default, i.e. `@//hw/bitstream:test_rom`?

Reply:
Okay to resolve the problem, I've changed the default bitstream value provided by `cw310()` to `None`. In `opentitan_functest`, if the `cw310` parameter is not provided or it's provided but no `bitstream` value is specified, then we will select the default bitstream based on the target type (`cw310_rom` vs `cw310_test_rom`). Otherwise, we use the default parameter. This should keep everything backwards compatible and allow us to specify a custom bitstream to override the default.

The only limitation I see is that we cannot use both a custom ROM and custom test ROM if we want to run the test with both the `cw310_rom` and `cw310_test_rom` targets. I don't think this will be needed, but if it is, we can modify this later to accept a list or dictionary mapping targets to desired bitstreams.